featuretools/tests/primitive_tests/test_expanding_primitives.py

```diff
@@ -133,3 +169,43 @@ def test_expanding_primitives_throw_warning(window_series_pd, primitive):
             numeric=window_series_pd,
             datetime=window_series_pd.index,
         )
+
+
+def test_apply_gap_for_expanding_primitives_throws_warning(window_series_pd):
+    with pytest.raises(TypeError):
```

Comment:
Since we are testing that we raise a custom TypeError here, we should also check that the error message matches what we expect. Without that, we don't know for sure that the error was the one we raised, or if it was coming from something else.